org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/doc/ApiDocGenerator.java

```diff
@@ -192,7 +195,7 @@ protected MethodDescriptor toMethodDescriptor(Method m) {
         .withMethod(m)
         .withHttpMethod(httpMethod)
         .withPath(path)
-        .withSignature(signature)
+        .withMethodSignature(methodSignature)
```

Comment:
According to other with-methods I wouldn't change this method name -> withSignature(methodSignature)

Reply:
i will change it.